src/include/sof/lib/pm_memory.h

```diff
@@ -0,0 +1,49 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Jakub Dabek <jakub.dabek@linux.intel.com>
+ */
+
+/**
+ * \file include/sof/lib/pm_memory.h
+ * \brief Runtime power management header file
+ * \author Jakub Dabek <jakub.dabek@linux.intel.com>
+ */
+
+#ifndef __SOF_LIB_PM_MEMORY_H__
+#define __SOF_LIB_PM_MEMORY_H__
+
+#include <sof/spinlock.h>
+#include <sof/trace/trace.h>
+#include <user/trace.h>
+#include <stdint.h>
+
+/** \brief Memory banks pm masks data. */
+struct ebb_data {
+	uint32_t ebb_current_mask0;
+	uint32_t ebb_current_mask1;
+	uint32_t ebb_new_mask0;
+	uint32_t ebb_new_mask1;
+	uint32_t change_mask0;
```

Comment:
No need to use ebb_ for any of these members, if you refer to this structure as ebb when in use i.e.
```c
ebb->current_mask0 = 0;
```